proto/c1/connector/v2/resource.proto

```diff
@@ -175,11 +175,12 @@ message CreateAccountResponse {
 
 message EncryptedData {
   string provider = 1;
-  string key_id = 2;
+  string key_id = 2 [deprecated = true];
   string name = 3;
   string description = 4;
   string schema = 5; // optional
   bytes encrypted_bytes = 6; // if 'schema' is set, this should be JSON.
```

Comment:
CreateAccountResponse contains a repeated EncryptedData, so why not just encrypt the data multiple times, once for each key id? That we can have the same data decryptable by multiple keys is an artifact of certain encryption schemes, not something that can be generalized.